SPECS/kernel/config

```diff
@@ -115,20 +119,21 @@ CONFIG_BPF_JIT=y
 CONFIG_BPF_JIT_ALWAYS_ON=y
 CONFIG_BPF_JIT_DEFAULT_ON=y
 CONFIG_BPF_UNPRIV_DEFAULT_OFF=y
+CONFIG_USERMODE_DRIVER=y
 # CONFIG_BPF_PRELOAD is not set
-# CONFIG_BPF_LSM is not set
+CONFIG_BPF_LSM=y
 # end of BPF subsystem
 
-CONFIG_PREEMPT_NONE=y
-# CONFIG_PREEMPT_VOLUNTARY is not set
+# CONFIG_PREEMPT_NONE is not set
+CONFIG_PREEMPT_VOLUNTARY=y
 # CONFIG_PREEMPT is not set
-# CONFIG_SCHED_CORE is not set
+CONFIG_SCHED_CORE=y
```

Comment:
allows cores to be shared https://www.kernel.org/doc/html/latest/admin-guide/hw-vuln/core-scheduling.html#:~:text=Core%20scheduling%20support%20is%20enabled%20via%20the%20CONFIG_SCHED_CORE,that%20can%20be%20co-scheduled%20on%20the%20same%20core.